src/sentry/integrations/issues.py

```diff
@@ -146,6 +146,15 @@ def make_external_key(self, data):
         """
         return data['key']
 
+    def get_issue_display_name(self, external_issue_key):
+        """
+        Returns the display name of the issue.
+
+        This is not required but helpful for integrations whose external issue key
+        does not match the disired display name.
+        """
+        return None
```

Comment:
Should this be an empty string to maintain type consistency with other implementations of this method?

Reply:
Hmm... so another js question: does empty string evaluate to False in js? If so it sounds like a pretty easy change to me.

Reply:
Yes. Empty strings are [falsy in javascript](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

```js
> if ('') { console.log('truthy') } else { console.log('falsy;') }
falsy;
```